cmd/gateway_validate.go

```diff
@@ -45,7 +42,11 @@ func executeValidate(cmd *cobra.Command, _ []string) error {
 	ctx := cmd.Context()
 	var kongClient *kong.Client
 	if validateOnline {
-		kongClient, err = getKongClient(ctx, targetContent)
+		if validateCmdRBACResourcesOnly && mode == modeKonnect {
+			return fmt.Errorf("rbac validation not yet supported in konnect mode")
```

Comment:
@GGabriele Added this condition here, as you mentioned that rbac stuff is not supported by Konnect. Let me know if error needs to be rephrased. Without this, users would get random errors/outputs when the flag is present.